src/EnumDefinition.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Spatie\Enum;
+
+class EnumDefinition
+{
+    /** @var string|int */
+    public $value;
```

Comment:
Don't make `public` properties make this class vulnerable for runtime changes?

Reply:
In principle, yes. Ever since PHP 7.4 we've been using public properties more often though. readonly properties are still missing unfortunately, but I feel it's not worth the overhead of adding getters, since this class is only to be used internally by the enum itself. That was my reasoning :)